js/apps/system/_admin/aardvark/APP/react/src/components/pure-css/form/Textinput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode, useEffect, useState } from 'react';
```

Comment:
What about the already existing Textbox? Could that be modified/parameterized instead to fit the use case for this?

Reply:
Will be resolved in https://github.com/arangodb/arangodb/pull/18199